irc/models.py

```diff
@@ -59,7 +60,7 @@ def parse(self, s, channel):
 
                 # check for ACTION:
                 if self.message.startswith(ACTION_START) and self.message.endswith(ACTION_END):
-                    self.message = self.message[8:]
+                    self.message = self.message[7:-1]
```

Comment:
Zakaj ta sprememba?

Reply:
Ko sem testiral, če je message tak kot bi moral biti, sem opazil da se prva črka sporočila ne izpiše, se pa izpiše tisti znak ACTION_END na koncu(kar je logično, saj je to tudi pogoj v if-u). Zato sem sklepal, da je bilo ponevedoma vse skupaj zamaknjeno za eno mesto nazaj. Potem sem pa pač popravil da res pride message tak kot smo ga vnesli. Razen če mogoče želimo izpisati tudi ACTION_START in ACTION_END (?), v tem primeru bi pa moralo pisati potem self.message[6:]